### ISSUE TITLE
The loss oscillates during training

### ISSUE THREAD

**Issue by 921418569** [alvinchangw/COCON_ICLR2021#10]
Hi. I'm sorry to bother you right now. I am a student now, and I am using cocon, but I met some difficulties in the training process. When I trained according to the train_cocon.sh provided by you, the final training loss was shock without convergence, I really don't know what went wrong, could you please help me, which is very important to me, thank you
